terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,17 @@
+
+# Account environment variables
+aws_profile = "development-eu-west-2"
+environment = "cidev"
+
```

Comment:
Looks like we don't need `kms_alias = "alias/development-eu-west-2/environment-services-kms"` here then?

Reply:
* Actually I see now this that @l-seviour removed this variable intentionally in [this commit](https://github.com/companieshouse/search.web.ch.gov.uk/commit/ab731c9bbbb5724b0a1e7d2e815a86979545e4bf). 
* Hence definitely no problem here. 👍